docs/adding_model/rnn.py

```
@@ -12,7 +12,12 @@ class RNNet(DLPredictionWrapper):
 
     def __init__(self, input_size, hidden_dim, layer_dim, num_classes, *args, **kwargs):
         super().__init__(
-            input_size=input_size, hidden_dim=hidden_dim, layer_dim=layer_dim, num_classes=num_classes, *args, **kwargs
+            input_size=input_size,
+            hidden_dim=hidden_dim,
+            layer_dim=layer_dim,
+            num_classes=num_classes,
+            *args,
```

Ruff (0.15.15)

[warning] Star-arg unpacking after a keyword argument is strongly discouraged

(B026)